datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMap.java

```diff
@@ -85,6 +85,11 @@
     this.analyzer = analyzer;
   }
 
+  /**
+   * default maximum records to return
+   */
+  private int max_doc = Integer.MAX_VALUE;
```

Comment:
I think a local variable is enough, not required to make it class member

Reply:
And change to `maxDoc`

Reply:
It seems there is no testcase for this parameter when user creating datamap, do I miss anything?